blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for Blog app."""
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Create Post model for Blog app."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish date method."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return title in admin."""
```

Comment:
This method is called when the object is being rendered into a string. Read https://docs.python.org/3/reference/datamodel.html for more details.
Don't state obvious things like "returns something" when it's literally a single instruction in this method and it's just one line below.

Reply:
How should I write correctly?

Reply:
Render post as a string.